mllib/src/test/scala/org/apache/spark/ml/feature/VectorAssemblerSuite.scala

```diff
@@ -126,4 +126,25 @@ class VectorAssemblerSuite
       .setOutputCol("myOutputCol")
     testDefaultReadWrite(t)
   }
+
+  test("VectorAssembler's UDF should not apply on filtered data") {
```

Comment:
mark the [SPARK-22446] on the test name.

Reply:
Ok.